server/lib/src/server/migrations.rs

```diff
@@ -933,6 +933,7 @@ impl<'a> QueryServerWriteTransaction<'a> {
             SCHEMA_ATTR_ACCOUNT_EXPIRE.clone().into(),
             SCHEMA_ATTR_ACCOUNT_VALID_FROM.clone().into(),
             SCHEMA_ATTR_API_TOKEN_SESSION.clone().into(),
+            SCHEMA_ATTR_APPLICATION_PASSWORD.clone().into(),
```

Comment:
see migrations comment above, this doesn't go here. It needs to be in the domain level 5 to 6 migration https://github.com/kanidm/kanidm/pull/2576